### PR TITLE
feat: add prettier and husky

### LINKED ISSUE #67: [Feature] Add prettier & husky to keep code style consistent
**Is your feature request related to a problem? Please describe.**
It would be nice to have formatter with well-defined rules and don't have to worry about what tools others use to format their code.

**Describe the solution you'd like**
Add prettier and optionally husky

### PR DESCRIPTION
**Pull request by ArturBa**
Fixes #67  

- [x] I have read and agree to the [Code of Conduct](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CONTRIBUTING.md).

## Pull Request Type

- [x] Bug fix (please link related issues if applicable)
- [ ] New feature
- [ ] Question update
- [ ] Other

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "*.tsx": ["prettier --write"],
```
Comment (masterHAWK99) [inline .lintstagedrc:2]: I can't test it now, but ensure `ts` files also format.

Comment (ArturBa) [reply .lintstagedrc]: fixed

**Review by eduardconstantin — COMMENTED** [review]
@masterHAWK99 if you consider this looks ok we can merge it.

File: .lintstagedrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "*.{jsx,js}": ["prettier --write"],
+  "*.{tsx,ts}": ["prettier --write"],
+  "*.{json,yml,md}": ["prettier --write"]
```
Comment (masterHAWK99) [inline .lintstagedrc:4]: You could put those in one line
```suggestion
  "*.{js,jsx,ts,tsx,json,md}": ["prettier --write"],
```

Comment (eduardconstantin) [reply .lintstagedrc:4]: also make sure tab space is at 2

Comment (ArturBa) [reply .lintstagedrc]: done

**Review by eduardconstantin — APPROVED** [review]
@masterHAWK99 looks good to me, if you approve I can merge it

**Review by masterHAWK99 — APPROVED** [review]

**Pull request merged**